include/benchmark/benchmark_api.h

```diff
@@ -230,7 +230,7 @@ BENCHMARK_UNUSED static int stream_init_anchor = InitializeStreams();
 // expression from being optimized away by the compiler. This function is
 // intended to add little to no overhead.
 // See: https://youtu.be/nXaxk27zwlk?t=2441
-#if defined(__GNUC__)
+#if defined(__GNUC__) && !defined(__pnacl__) && !defined(EMSCRIPTEN)
```

Comment:
Does this actually fail to compile? If so that's a Clang bug. The inline assembly doesn't actually generate instructions, so I would hope it would still work regardless of Clang's target.

If not can you provide a full error log so I can used it to file a Clang bug?